skore/src/skore/sklearn/_plot/base.py

```diff
@@ -10,3 +12,15 @@ def plot(self, **kwargs: Any) -> None:
 
     def set_style(self, **kwargs: Any) -> None:
         """Set the style of the display."""
+
+    def frame(self) -> pd.DataFrame:
+        """Get the data used to create the display.
+
+        The returned DataFrame structure depends on the type of plot (ROC curve,
+        precision-recall curve, prediction error, etc.).
+
```

Comment:
Since it is a general protocol I don't think that we need to mention anything here. Let's keep it short.
```suggestion
```